### DISCUSSION TITLE
chunks of dynamic imports with systemjs

### DISCUSSION THREAD

**Discussion by jefferson-william** [webpack/webpack#13893] [Ideas]
When I set `config.output.libraryTarget = 'system'` and some chunk of a webpack dynamic import is created, it doesn't contain systemjs.

That would be an awesome microfrontend feature.

**Comment by jefferson-william** [comment]
Related discussion:

- https://github.com/systemjs/systemjs/issues/2308

**Reply by alexander-akait** [reply]
Can you provide example of the problem?
